box.json

```diff
@@ -27,7 +27,8 @@
         "coldbox":"^6.0.0",
         "commandbox-dotenv":"*",
         "commandbox-cfconfig":"*",
-        "commandbox-migrations":"*"
+        "commandbox-migrations":"*",
+        "cbSwagger":"^2.6.0+147"
```

Comment:
Can you remove this. This is already included in the `modules/contentbox/box.json`